tsconfig.json

```diff
@@ -100,6 +100,5 @@
         "skipLibCheck": true /* Skip type checking all .d.ts files. */
     },
     "types": ["color", "events"],
-    "include": ["./src/**/*.ts", "./example/src/**/*.ts"],
-    "exclude": ["./src/**/*.spec.ts"]
```

Comment:
why are test files now included in the prod tsconfig?

Reply:
because lint complained - that they were not part of the tsconfig project.

Reply:
won't that cause tsc to also compile test files when building for production?

Reply:
yeah it does - ... looking

Reply:
remvoed

Reply:
eslint was using tsconfig.eslint.json before, which includes spec files also
not sure what is the issue here, but it worked before and nothing complained

tsconfig.json was used for prod - excluded spec files
tsconfig.spec.json was used for spec files and only included spec files
tsconfig.eslint.json was used for eslint and included everything

which IDE are you using? and can you post the lint error you are facing?